codec-http/src/main/java/io/netty/handler/codec/http/HttpContentEncoder.java

```diff
@@ -113,9 +112,10 @@ protected void encode(ChannelHandlerContext ctx, HttpObject msg, List<Object> ou
                 final HttpResponse res = (HttpResponse) msg;
                 final int code = res.status().code();
                 final CharSequence acceptEncoding;
-                if (code == CONTINUE_CODE) {
-                    // We need to not poll the encoding when response with CONTINUE as another response will follow
-                    // for the issued request. See https://github.com/netty/netty/issues/4079
+                if (code >= 100 && code < 200) {
```

Comment:
To avoid hardcoding constants, consider checking that `res.status().codeClass() == INFORMATIONAL`.

Reply:
Done